dynatrace-operator-google-marketplace/charts/dynatrace-activegate-operator/templates/secret.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright 2019 Dynatrace LLC
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+{{- if and (.Values.apiToken) (.Values.paasToken) }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Values.name }}
+  namespace: {{ .Release.Namespace }}
+data:
+  apiToken: {{ .Values.apiToken | b64enc }}
+  paasToken: {{ .Values.paasToken | b64enc }}
```

Comment:
Doesn't the proxy needs to be here as well?

Reply:
Added